tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationView.java

```diff
@@ -284,7 +285,17 @@ else if( url.startsWith( "http" ) )
         {
             public void selectionChanged( SelectionChangedEvent event )
             {
-                List<TaskProblem> problems = MigrationUtil.getTaskProblemsFromTreeNode( event.getSelection() );
+                List<TaskProblem> problems;
```

Comment:
We wont need this code once the logic is moved into getTaskProblemsFromTreeNode()